bin/peas_shell

```diff
@@ -216,16 +219,35 @@ class PanSensorShell(cmd.Cmd):
             'cam_1': {'pin': 6, 'board': 'camera_board'},
         }
 
+        # NOTE: These are not pins but index numbers
```

Comment:
This is why I'd like to have each board tell us it's set of relays as the names in the relays object in the JSON report.

Reply:
Agreed. I think during the `setup` we could output some info about the board. Or just deliver it along with each report. This means you would have to be good about having the correct plug in each relay but that's probably a good idea anyway.